features/step_definitions/description_change_steps.rb

```diff
@@ -2,10 +2,6 @@
 
 require "faker"
 
-Given("a determined planning application") do
-  @planning_application.update!(status: "determined", determined_at: Time.zone.now)
```

Comment:
for the record (no pun intended), this creates a wrong determined application: because the status is updated by bypassing AASM, it's hard to notice that the resulting object is a determined application without a decision, which doesn't make any sense. 

"There should be a guard for that!" - [yes](https://github.com/unboxed/bops/blob/master/app/models/planning_application.rb#L66) but it's bypassed.